nix/cells/automation/pipelines.nix

```diff
@@ -27,13 +25,15 @@ in
   ci = { config, lib, ... }: {
     imports = [ common ];
 
-    preset.github-ci.sha = config.preset.github-ci.lib.readRevision
+    preset.github.ci.revision = config.preset.github.lib.readRevision
       inputs.cells.cloud.library.actions.ci.input
       null;
 
-    command.text = ''
-      nix build -L .#${lib.escapeShellArg system}.automation.ciJobs.required
-    '';
+    command.text = config.preset.github.status.lib.reportBulk {
+      bulk.text = "nix eval .#__std.init --apply __attrNames --json | nix-systems -i";
```

Comment:
What does this do? What is `nix-systems`? Shouldn't we be splicing that in, how do we know it's going to be present on the system? Implicit runtime dependencies make us sad...

Reply:
[`nix-systems`](https://github.com/input-output-hk/tullia/blob/ead1f515c251f0e060060ef0e2356a51d3dfe4b0/cells/tullia/apps.nix#L229) is added to the runtime dependencies by `preset.nix.enable = true`.

```
❯ nix run github:input-output-hk/tullia#nix-systems -- -h
Without arguments, prints which systems nix can build for on this machine.
With -l, considers only the system of local machine to be supported.
With -r, considers only systems of remote builders to be supported.
With -i, prints whether the systems read from the JSON list on stdin are supported.
With -I, prints whether the systems read from lines on stdin are supported.
With -O, prints supported systems line by line.
With -o, prints unsupported systems line by line.
```